src/main/flight/mixer.c

```diff
@@ -721,12 +722,37 @@ static void applyFlipOverAfterCrashModeToMotors(void)
     }
 }
 
+static float applyThrustLinearization(float motorOutput)
+{
+#ifdef USE_THRUST_LINEARIZATION
+    if (motorConfig()->thrustLinearization) {
+        float tl = motorConfig()->thrustLinearization * 0.01f;
+        static float a, a_reci, b, b_sq;
+        if (a != tl) {
+            a = tl;
+            a_reci = 1.0f / a;
+            b = (1.0f - a) / (2.0f * a);
+            b_sq = b * b;
+        }
+
+        if (motorOutput > 0.0f) {
+            motorOutput = 1.0f / fast_rsqrt(motorOutput * a_reci + b_sq) - b;
```

Comment:
Add brackets for readability?
```motorOutput = (1.0f / fast_rsqrt((motorOutput * a_reci) + b_sq)) - b;```

Reply:
To my eyes the extra brackets make it less readable, but I'm open to change it if more people find your suggestion more appealing.

Reply:
Yeah, the coding style document isn't really up to date there with what we are practicing in Betaflight these days. Opened #7312 to update this.